src/v/cloud_storage/partition_manifest.cc

```diff
@@ -1206,7 +1206,7 @@ void partition_manifest::spillover(const segment_meta& spillover_meta) {
           cst_log.error,
           "{} Expected spillover metadata {} doesn't match actual spillover "
           "metadata {}",
-          _ntp,
+          get_manifest_path(),
```

Comment:
the message isn't part of this change, but wdyt of making this `[{}]` to be more uniform?

Reply:
good point